app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -11,7 +12,7 @@
     <label class="govuk-label" for="<%= id %>">
       <%= label %>
     </label>
-    <select class="govuk-select" id="<%= id %>" name="<%= id %>" <%= data_module %> >
+    <select class="govuk-select <%= 'gem-c-select__select--full-width' if full_width %>" id="<%= id %>" name="<%= id %>" <%= data_module %> >
```

Comment:
Bikeshedding, so truly actually just a suggestion. 

The components seem inconsistent with each other with how classes are built up - but the joy is that it all works. 

Can we set the class as a variable, the push the `gem-c-select__select--full-width` class with an if? Any future extra classes can then be added on by `<<` on a new line.

```ruby
class = "govuk-select"
class << "gem-c-select__select--full-width" if full_width
```

Reply:
I tend to use that technique if there's more than two classes, because it gets unreadable at that point. But I think for two it's probably okay.